### ISSUE TITLE
[BUG] Duchess dress deletes hair when facing south

### ISSUE THREAD

**Issue by wow50000** [Rotwood-Vale/Ratwood-Keep#1592]
## Describe the bug
![image](https://github.com/user-attachments/assets/2d1422df-12b9-4e47-b2e4-8cb8d9e5af8d)
with dress
![image](https://github.com/user-attachments/assets/2b1ea482-47ff-47af-ac1f-a0ff10847898)
without dress
I want hair to overlay over the dress when facing south due to how it is positioned behind the head

## Steps to Reproduce: 
Wear the duchess dress, face south

**Comment by TheFenix323** [comment]
Unable to replicate this on local 
![image](https://github.com/user-attachments/assets/ae70bb37-fbd7-4843-9a58-bc3e2fda46a2)

**Comment by ScienceMom** [comment]
Yeah, for some reason a lot of these new Court clothes are buggy. Missing sleeves when facing sideways, missing hair, ect ect ect. I might take a look at this.
